hpy/devel/include/hpy/hpydef.h

```diff
@@ -145,7 +145,7 @@ typedef struct {
 
 #define HPyDef_METH_IMPL(SYM, NAME, IMPL, SIG, ...)                     \
     HPyFunc_DECLARE(IMPL, SIG);                                         \
-    HPyFunc_TRAMPOLINE(SYM##_trampoline, IMPL, SIG);                    \
+    HPyFunc_TRAMPOLINE(SYM##_trampoline, IMPL, SIG)                     \
```

Comment:
Yes, that's on purpose. I sometimes add `-pedantic` to tests compilation to spot possible problems and this was one of the warnings since the macro expands to several statements and this semicolon is not necessary.
But I don't have  a strong opinion. If you want to keep it, I can re-introduce it but I would like to do it in a follow-up PR to save CI time.